elasticsearch/README.md

```diff
@@ -0,0 +1,9 @@
+# Build Elasticsearch image
+```sh
+$ docker build -t es .
+```
+
+# Run Elasticsearch Container
+```sh
+$ docker run -d -p 9200:9200 -p 9300:9300 -v /home/docker/elasticsearch:/data es -Des.config=/data/elasticsearch.yml -Des.default.path.logs=/data 
```

Comment:
You dont need to provide elasticsearch.yml location and logs location in the docker run. Instead modify your Dockerfile's entry point and provide them.